src/middlewared/middlewared/plugins/system/coredump.py

```diff
@@ -14,15 +14,20 @@ def coredumps(self):
             with journal.Reader() as reader:
                 reader.add_match(CODE_FUNC='submit_coredump')
                 for core in reader:
-                    coredump = {
-                        'time': core['COREDUMP_TIMESTAMP'].strftime('%c'),
-                        'pid': core['COREDUMP_PID'],
-                        'uid': core['COREDUMP_UID'],
-                        'gid': core['COREDUMP_GID'],
-                        'unit': core['COREDUMP_UNIT'],
-                        'sig': core['COREDUMP_SIGNAL'],
-                        'exe': core['COREDUMP_EXE'],
-                    }
+                    try:
+                        coredump = {
+                            'time': core['COREDUMP_TIMESTAMP'].strftime('%c'),
+                            'pid': core['COREDUMP_PID'],
+                            'uid': core['COREDUMP_UID'],
+                            'gid': core['COREDUMP_GID'],
+                            'unit': core.get('COREDUMP_UNIT', '<UNKNOWN>'),
```

Comment:
I don't see any consumers that expect a string here, might as well just be `None`